### PR TITLE
[FIX] web_editor: display grid mode mobile view for higher screen sizes

### PR DESCRIPTION
**Pull request by sobo-odoo**
When we are in grid mode, if we decrease the screen size small enough, the display is back to `flex` and the columns are displayed one above another (-> mobile view). However, the columns in normal mode do that for screen sizes higher than the breakpoint set for the ones in grid mode, which is inconsistent. Moreover, as the images have their `object-fit` property set to `cover`, they become really deformed and therefore do not look good.

This commit solves these issues by increasing the breakpoint at which the grid mode switches to the mobile view, that is, from `md` to `lg`.

opw-3069234

### CODE REVIEW

**Comment by sobo-odoo** [comment]
@qsm-odoo @rdeodoo 
FYI: I also had to modify in snippets.options where I check if the mobile view is activated to update the overlay, since it is based on the screen size.

File: addons/web_editor/static/src/js/editor/snippets.options.js [hunk 1]
```diff
@@ -3791,7 +3791,7 @@ const SnippetOptionWidget = Widget.extend({
         if (moveUpOrLeft || moveDownOrRight) {
             // The arrows are not displayed if the target is in a grid and if
             // not in mobile view.
-            const isMobileView = this.$target[0].ownerDocument.defaultView.frameElement.clientWidth < 768;
+            const isMobileView = this.$target[0].ownerDocument.defaultView.frameElement.clientWidth < 992;
```
Comment (qsm-odoo) [inline addons/web_editor/static/src/js/editor/snippets.options.js:3794]: @sobo-odoo Could you do better and use the utils to do this? config.js (legacy) / ui_service.js (new)? :+1: Ideally there are ways to write something like "if screenSize below or equal to MD", without re-hardcoding numbers.

**Review by qsm-odoo — CHANGES_REQUESTED** [review]
Commit message "unconsistent" -> "inconsistent"

**Comment by qsm-odoo** [comment]
@sobo-odoo After more testing and thinking with PDE and RDE, we decided to add another option for grid images to be able to switch between cover and contain. I quickly made a commit (added on top of this PR): could you review it, test it and tell me if you think something is not right about this? Thanks :+1:

**Review by qsm-odoo — APPROVED** [review]
Thanks for the review @sobo-odoo (typos + the image shape stuff) :+1: I actually also forgot to add the new translated terms to the .pot.

Both commits are not really stable but we decided that it was worth it compared to the different alternatives, keeping in mind the 16.0 is recent:
- Mobile view is triggered on higher breakpoint than before for grid mode (~= portrait tablet), like for the non-grid mode. This will probably solves more usecases than it breaks (if any).
- Images using shape in grid mode now behave differently (including existing ones). Should be minimal as only concerns images whose area was reduced to a smaller height than the image itself.

@robodoo rebase-ff r+

**Pull request closed without merging**